tools/InterfaceGenerator/generator/parsers/RPCBase.py

```diff
@@ -80,6 +80,9 @@ def _initialize_enums(self):
             [("FunctionID", Model.Enum(name="FunctionID")),
              ("messageType", Model.Enum(name="messageType"))])
 
+    def _check_function(self, enum):
```

Comment:
What is the purpose of this?

Reply:
@jacobkeeler this is interface function, that is overridden in MOBILE and HMI RPC generators.